docs/react-wiki-archive/API-Documentation.md

```diff
@@ -0,0 +1,136 @@
+# Documenting Fluent UI
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31202&commitId=aa7daf9fa48826386298f962710c8d2aa3271293&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.basic - High Contrast.normal.chromium.png|0| Added|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Image Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image Converged.Image Variations Fallback.normal.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>